docs/reference/themes/mongodb/layouts/partials/assets/javascripts.html

```diff
@@ -13,3 +13,4 @@
 <script type="text/javascript" src="{{.Site.BaseURL}}/lib/highlight/highlight.pack.js"></script>
 <script type="text/javascript" src="{{.Site.BaseURL}}/js/scripts.js"></script>
 {{ partial "assets/javascriptExtras.html" . }}
+<script type="text/javascript" src="{{.Site.BaseURL}}/lib/delighted.js"></script>
```

Comment:
nit: lets add this above the partial

Reply:
Also should we just load the delighted.js from the home page?

```
<script type="text/javascript" src="{{.Site.BaseURL}}/../s/js/delighted.js"></script>
```

That means in gh-pages we could just replace all previous versions of :

```
<script type="text/javascript" src="{{.Site.BaseURL}}/js/scripts.js"></script>
```

with 

```
<script type="text/javascript" src="{{.Site.BaseURL}}/js/scripts.js"></script>
<script type="text/javascript" src="{{.Site.BaseURL}}/../s/js/delighted.js"></script>
```

Should be easier to update all the versions of the documentation then.

Reply:
@jstewart-mongo If this is proving to hard to test we can still go with two versions of the script, one in landing and one in reference. Then you can test locally.

Reply:
I think it will be safer to have two versions of the script for testing.

Reply:
I have moved the line above the partial.